src/test/java/Homework19.java

```diff
@@ -0,0 +1,40 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework19 extends BaseTest {
+
+    @Test
+    public void deletePlaylistTest() throws InterruptedException {
+
+        login("pooja.bankar@testpro.io","te$t$tudent");
+        clickSubmit();
+        selectExistingPlaylist();
+        selectDeletePlaylist();
+        Assert.assertTrue(isPlaylistDeletedMessageDisplayed());
+      
+
+    }
+
+    //Selecting existing playlist
+    public void selectExistingPlaylist() throws InterruptedException {
+        WebElement existingPlaylist = driver.findElement(By.xpath("//*[@id=\"playlists\"]/ul/li[3]"));
+        existingPlaylist.click();
+        Thread.sleep(2000);
+    }
+
+    //Selecting Delete Playlist Option
+    public void selectDeletePlaylist() throws InterruptedException {
+        WebElement deletePlaylistOption = driver.findElement(By.cssSelector("button.del.btn-delete-playlist"));
+        deletePlaylistOption.click();
+        Thread.sleep(2000);
```

Comment:
Using Thread. sleep() frequently in an automation framework is not a good practice.
Try to use it as less as you can - 
Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.